automatic/selenium.powershell/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,51 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = Split-Path -Parent $MyInvocation.MyCommand.Definition
+
+$ModuleName = "Selenium"
```

Comment:
Using an uppercase for the first letter goes against the practices that have already been present in this repository for years.

Please update the variables used to be consistent with existing code style everywhere.